controllers/drpolicy_controller.go

```diff
@@ -243,6 +250,34 @@ func s3ProfileValidate(ctx context.Context, apiReader client.Reader,
 	return "", nil
 }
 
+func validateCIDRsFormat(drpolicy *ramen.DRPolicy) error {
+	// validate the CIDRs format
+	for i := range drpolicy.Spec.DRClusterSet {
+		cluster := &drpolicy.Spec.DRClusterSet[i]
+		if err := CIDRsFomatValidate(cluster.CIDRs); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func CIDRsFomatValidate(cidrs []string) error {
+	invalidCidrs := []string{}
+
+	for i := range cidrs {
+		if _, _, err := net.ParseCIDR(cidrs[i]); err != nil {
```

Comment:
the actual error message should be returned or else difficult to identify what is wrong?

Reply:
To add:
We log errors from external functions, and pass along errors from internal functions upwards. This helps log where we faced an error from an external routine. e.g [here](https://github.com/RamenDR/ramen/blob/ff29bf0d3dbbcd116dc683c3e91d7152e22e8d3b/controllers/volumereplicationgroup_controller.go#L706-L711)

CIDR validation is pretty basic, IOW it is not invoking a request/response like routine, and hence we can short circuit logging in one place, as below where we return the error.

So to address this, please add a log message above the error return (line 275), that logs all invalid CIDRs found, and return a list of invalid CIDRs with a message in the error, say "failed to parse CIDRs %s". 

Now this error message does not yet tell us which drCluster configuration is invalid, and this error message would be present in the status of the DRPolicy reconciler. To fix, in order to make it easier for the user to see the actual error, in the `ValidateCIDRsFormat` function further add required message on which cluster set the invalid CIDRs were found and return that as the error.

Reply:
Added logging err from `net.ParseCIDR` and also added cluster name in the return error message.